tools/Godot.AppDir/AppRun

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+HERE=$(dirname $(readlink -f "${0}"))
+export LD_LIBRARY_PATH="${HERE}"/usr/lib:$PATH
+"${HERE}"/usr/bin/godot $@
```

Comment:
This is a bit convoluted as we don't need a `LD_LIBRARY_PATH`, I think you could just drop the `godot` binary in the main folder and run it directly.